Source/EventParser.swift

```diff
@@ -64,15 +70,20 @@ class EventParser {
     }
 
     private func dispatchEvent() {
-        guard !data.isEmpty
```

Comment:
I think this was a change for in the spec, but I'm not certain. Basically the difference is whether we consider:
```
id: 4

```
To be a complete event and update our stored last event ID to be used on the next connection, or whether we wait for message with `data: ` to complete.

Reply:
I think that that description of [the spec](https://html.spec.whatwg.org/multipage/server-sent-events.html) is slightly wrong, in that if there is no `data` then there is no "complete event"... but the updated code here is correct. That is, if there is no `data` then we do not actually dispatch an event, but we can still update `lastEventId`.

Reply:
Right, my wording here wasn't quite ideal. It's still not an event that we dispatch when we've received no data, but we consider it to be a valid update to the `lastEventId`.